.golangci.yml

```
@@ -84,11 +83,14 @@ issues:
         - ineffassign
         - govet
         - typecheck
+    - path: main.go
+      linters:
+        - typecheck
   new: false
 
 # golangci.com configuration
 # https://github.com/golangci/golangci/wiki/Configuration
 service:
-  golangci-lint-version: 1.43.x # use the fixed version to not introduce new linters unexpectedly
+  golangci-lint-version: 1.52.x # use the fixed version to not introduce new linters unexpectedly
```

Need to match the version in the makefile
```suggestion
  golangci-lint-version: 1.53.x # use the fixed version to not introduce new linters unexpectedly
```